### PR TITLE
Added Ephemeral Execute with Tag option

### PR DESCRIPTION
**Pull request by grafjo**
cli part for https://github.com/carbynestack/ephemeral/pull/17

### CODE REVIEW

**Comment by grafjo** [comment]
@sbckr have rebased to current workflow files

File: src/main/java/io/carbynestack/cli/client/ephemeral/command/ExecuteEphemeralClientCliCommandRunner.java [hunk 1]
```diff
@@ -42,15 +43,29 @@ private Either<Throwable, String> readCodeFromStdIn() {
...
+    }
+
     String code =
         readCodeFromStdIn()
             .getOrElseThrow(
                 e ->
                     new CsCliRunnerException(
                         getMessages().getString("execute.failure.read-code"), e));
+
+    Future<Either<ActivationError, List<ActivationResult>>> execute;
```
Comment (sbckr) [inline src/main/java/io/carbynestack/cli/client/ephemeral/command/ExecuteEphemeralClientCliCommandRunner.java:60]: I would rather call it execution, but that is a personal preference.

File: src/test/java/io/carbynestack/cli/client/ephemeral/EphemeralClientCliTest.java [hunk 2]
```diff
@@ -153,6 +153,31 @@ public void givenInputOptionWithInvalidUUID_whenParsing_thenExceptionWithCorrect
...
   }
 
+  @Test
+  public void
+      givenBothTagFiltersAndInputsProvided_whenUsingEphemeralClient_thenExceptionWithCorrectMessageIsThrows()
+          throws CsCliLoginException, CsCliException {
+    String uuid = UUID.randomUUID().toString();
+    String tagFilter = "key:value";
+
+    try {
```
Comment (sbckr) [inline src/test/java/io/carbynestack/cli/client/ephemeral/EphemeralClientCliTest.java:163]: I am aware that the existing tests for exceptions were all implemented with try/catch, but I would prefer the transition to Assert.assertThrows. Like

```java
CsCliRunnerException actualCCRE = Assert.assertThrows(CsCliRunnerException.class, () ->
      getCliWithArgs(
              ExecuteEphemeralClientCliCommandConfig.COMMAND_NAME,
              "-i",
              uuid,
              "-f",
              tagFilter,
              "app.example.com")
          .parse());
    assertEquals(actualCCRE.getMessage(),
            EPHEMERAL_CLI_MESSAGES.getString("execute.failure.both-inputs-provided"));
```

**Review by sbckr — CHANGES_REQUESTED** [review]
PR is missing dependency update for the Ephemeral java client (pom file) which requires https://github.com/carbynestack/ephemeral/pull/17 to be merged

**Comment by sbckr** [comment]
I have just realised, that the newly added option is not yet described in the readme`. (see [README.md#usage](https://github.com/carbynestack/cli#usage)).

So far we have inserted the respective help output at the corresponding place

**Comment by sbckr** [comment]
I have just realised, that the newly added option is not yet described in the readme`. (see [README.md#usage](https://github.com/carbynestack/cli#usage)).

So far we have inserted the respective help output at the corresponding place

**Comment by strieflin** [comment]
Needs to be addressed in a holistic way (see carbynestack/amphora#34).

**Pull request closed without merging**